[2021-07-02 05:38:47,795] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.print_date 2020-06-01T05:55:00+00:00 [queued]>
[2021-07-02 05:38:47,806] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.print_date 2020-06-01T05:55:00+00:00 [queued]>
[2021-07-02 05:38:47,807] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 05:38:47,807] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-07-02 05:38:47,808] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 05:38:47,821] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): print_date> on 2020-06-01T05:55:00+00:00
[2021-07-02 05:38:47,828] {standard_task_runner.py:52} INFO - Started process 148639 to run task
[2021-07-02 05:38:47,831] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'print_date', '2020-06-01T05:55:00+00:00', '--job-id', '380', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpjv0_t7ym', '--error-file', '/tmp/tmpou8e7x7w']
[2021-07-02 05:38:47,833] {standard_task_runner.py:77} INFO - Job 380: Subtask print_date
[2021-07-02 05:38:47,881] {logging_mixin.py:104} INFO - Running <TaskInstance: test.print_date 2020-06-01T05:55:00+00:00 [running]> on host 9b3ed935c8bc
[2021-07-02 05:38:47,946] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@airflow.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2020-06-01T05:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-06-01T05:55:00+00:00
[2021-07-02 05:38:47,947] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-02 05:38:47,947] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'date']
[2021-07-02 05:38:47,960] {subprocess.py:75} INFO - Output:
[2021-07-02 05:38:47,962] {subprocess.py:79} INFO - Fri Jul  2 05:38:47 UTC 2021
[2021-07-02 05:38:47,963] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-02 05:38:47,990] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=test, task_id=print_date, execution_date=20200601T055500, start_date=20210702T053847, end_date=20210702T053847
[2021-07-02 05:38:48,023] {taskinstance.py:1245} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-07-02 05:38:48,047] {local_task_job.py:151} INFO - Task exited with return code 0
